src/include/daos_srv/bio.h

```diff
@@ -1054,6 +1054,28 @@ struct bio_wal_info {
  */
 void bio_wal_query(struct bio_meta_context *mc, struct bio_wal_info *info);
 
+/* WAL replay stats */
+struct bio_wal_rp_stats {
+	void		*wrs_metrics;	/* pointer to module metrics */
+	uint64_t	wrs_tm;		/* rehydration time */
+	uint64_t	wrs_sz;		/* bytes replayed */
+	uint64_t	wrs_entries;	/* replayed entries count */
```

Comment:
Yes, return all 'stats' required for metrics, and this structure should be defined & exported by umem.